076-moving-data-between-two-jbod-disks-using-cruise-control.md

```diff
@@ -0,0 +1,85 @@
+# Moving data between two JBOD disks using Cruise Control
+
+This proposal is about integrating the [`remove_disks`](https://github.com/linkedin/cruise-control/blob/main/cruise-control/src/main/resources/yaml/endpoints/removeDisks.yaml) endpoint from Cruise Control into Strimzi cluster operator. 
+This endpoint will allow us to move the data between two JBOD disks. 
+
+## Current situation
+
+Currently, we get a multiple requests from community users to add the ability for moving all Kafka logs between two disks on the JBOD storage array. This feature can be useful in following scenarios:
+- The current disk is too big and the user wants to use a smaller one and vice versa is true.
+- When we want to use a different Storage Class with different parameters or different storage types.
+- In case of disk removal to reduce the total storage.
+
+For now, we can do this using the Kafka CLI tool i.e. `kafka-reassign-partitions.sh` tool, but it takes a lot of manual steps which is time-consuming and not so user-friendly.
+
+## Motivation
+
+We should introduce the logic to Strimzi to leverage Cruise Control integration and make it possible to move the data between two JBOD disks.
+This feature will also allow us to remove the disks without the loss of data.
+
+## Proposal
+
+Cruise Control provides `remove_disks` HTTP REST endpoint to move replicas from a specified disk to other disks of the same broker.
+This endpoint triggers a rebalancing operation by moving replicas in a size-based manner to the remaining disks, from the largest to the smallest, while checking the following constraint:
```

Comment:
So, this is always intra-broker rebalancing not moving any data between brokers?

Reply:
Yes, its always intra-broker rebalancing.

Reply:
Ok, thanks. Maybe its worth clarifying it in the proposal and in the future docs?